solv_map.hpp:332:20: error: no member named 'out_of_range' in namespace 'std'

```
In file included from /usr/ports/sysutils/dnf5/work/dnf5-5.1.0/libdnf5/advisory/advisory_sack.cpp:20:
In file included from /usr/ports/sysutils/dnf5/work/dnf5-5.1.0/libdnf5/advisory/advisory_sack.hpp:23:
/usr/ports/sysutils/dnf5/work/dnf5-5.1.0/libdnf5/./solv/solv_map.hpp:332:20: error: no member named 'out_of_range' in namespace 'std'
        throw std::out_of_range("Id is out of bitmap range");
              ~~~~~^
```

```std::out_of_range``` requires ```#include <stdexcept>``` which isn't present in ```solv_map.hpp```.

Version: 5.1.0
clang-15
FreeBSD 13.2

--- a/libdnf5/solv/solv_map.hpp
+++ b/libdnf5/solv/solv_map.hpp
@@ -26,6 +26,7 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include <solv/pooltypes.h>
 
 #include <iterator>
+#include <stdexcept>
 
 
 namespace libdnf5::solv {

